cmd/algoh/main.go

```diff
@@ -60,12 +60,13 @@ func (c *stdCollector) Write(p []byte) (n int, err error) {
 }
 
 func main() {
+	blockWatcherInitialized := false
 	flag.Parse()
 	nc := getNodeController()
 
 	genesisID, err := nc.GetGenesisID()
 	if err != nil {
-		fmt.Fprintln(os.Stdout, "error loading telemetry config", err)
+		_, _ = fmt.Fprintln(os.Stdout, "error loading telemetry config", err)
```

Comment:
What's the purpose of the "_, _"?  Were you getting lint errors or something?

Reply:
OK, Goland required to get the green check for code analysis.  I will remove.